lib/tasks/migrate_to_pg.rake

```diff
@@ -92,3 +107,24 @@ def migrate_model(pg_model, mongo_model, transform = nil)
 
   puts "#{pg_model.count} #{pg_model} migrated to pg"
 end
+
+def migrate_fulfillment(pt_model, mongo_pt_model, pg_model, mongo_model, relation, parent_relation, transform)
```

Comment:
is this in prep for reuse on archive patient?

Reply:
yep! Because fulfillment is 1-1 rather than 1-many, we know we're always porting over one object, so we write a specialized handler here that doesn't iterate.

Side note,  when we're all done with this we should consider just slamming the fields onto the patient object rather than dealing with them separately. The separate fulfillment object feels increasingly silly. Not something we

Reply:
not something we?

I don't have strong feelings on subobj vs embedded!

Reply:
Pull request experimental fiction 😂 

Not something we have to resolve in the scope of this of course. But when I was working on it I was like 'this reminds me of the original model mistake we made, trying to shoehorn pregnancy data into a separate object'. (though fulfillment is a much cleaner separation and a lot less conceptually hard.)